src/marker.rs

```diff
@@ -106,7 +163,7 @@ impl MarkerListener {
         )
         .unwrap();
 
-        MarkerListener {
+        MarkerArrayListener {
```

Comment:
What happened to the marker listener?

Reply:
the diff is a little broken, i renamed the old MarkerListener to MarkerArrayListener (since that it what it does). Its clearer when looking at the whole file